opts/fixtures/envdir/trim/FOO

```diff
@@ -0,0 +1 @@
+  bar   
```

Comment:
Since we ignore 2+ lines, go ahead and add a comment saying that there are spaces after 'bar' on the first line so that people don't erase them by mistake.